src/GridInterpolations.jl

```diff
@@ -108,11 +108,11 @@ function Base.show(io::IO, grid::AbstractGrid)
     end
 end
 
-function ind2x(grid::AbstractGrid, ind::Int)
+function ind2x(grid::AbstractGrid{D}, ind::Int) where D
     ndims = dimensions(grid)
-    x = Array{Float64}(undef, ndims)
+    x::MVector{D, Float64} = @MVector zeros(D)
     ind2x!(grid, ind, x)
-    x::Array{Float64}
+    x::MVector{D, Float64}
```

Comment:
why do you need the type annotation here? I would check with `@code_warntype` that this is type stable.

Reply:
I just kept the type annotation that was already present in the form of `::Array{Float64}`, but should not be necessarry, agreed. My understanding was that it's more explicit than required, but also doesn't harm, right?